src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const basePrice = days * PRICE;
+
+  switch (true) {
+    case days >= LONG_TERM:
+      return basePrice - LONG_TERM_DISCOUNT;
+    case days >= SHORT_TERM:
+      return basePrice - SHORT_TERM_DISCOUNT;
+    default:
+      return basePrice;
+  }
 }
```

Comment:
The function `calculateRentalCost` is well-named and the constants are descriptive and follow the checklist. However, consider refactoring the switch statement into if-else statements for better readability and to adhere to the checklist's preference for `if` with `return` over `if else`.